update_buildscript.sh

```diff
@@ -1,5 +1,11 @@
-#!/bin/bash
+#!/usr/bin/env bash
 curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/build.gradle -o build.gradle -sS && echo "Build.gradle downloaded"
 curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/.github/workflows/release-tags.yml -o .github/workflows/release-tags.yml -sS --create-dirs && echo "Release Tags action downloaded" 
 curl https://raw.githubusercontent.com/GTNewHorizons/ExampleMod1.7.10/main/CODEOWNERS -o CODEOWNERS -sS && echo "CODEOWNERS downloaded"
-git update-index --chmod=+x gradlew && git add CODEOWNERS .github/workflows/release-tags.yml build.gradle && echo "Everything added to git"
+rm -f .github/scripts/test-no-crash-reports.sh
```

Comment:
nit: Doesn't curl overwrite? Or it can with an option?

Reply:
the name changes from test-no-crash-reports (old buildscripts) to test-no-error-reports

Reply:
ah, makes sense.